src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,57 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Robot;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.IntakeSubsystem;
+
+/**
+ * An example command.  You can replace me with your own command.
+ */
+public class IntakeCommand extends CommandBase {
+  public IntakeCommand() {
+    // Use requires() here to declare subsystem dependencies
+    requires(Robot.intake);
+  }
+
+  private void requires(IntakeSubsystem intake) {
+}
+
+// Called just before this Command runs the first time
+  @Override
+  public void initialize() {
+  }
+
+  // Called repeatedly when this Command is scheduled to run
+  @Override
+  public void execute() {
```

Comment:
uhhhh is this just from last year 
our intake this year will only ever spin one way

Reply:
OK I know this probably hasn't been what you were actually working on, it was just in your code.

Talk to me when you see this, I can prob push this up and have someone else/me change the subsystem stuff while you continue with vision stuff.

Reply:
I see it, and that's fine, it doesn't bother me